frameworks-versions.properties

```diff
@@ -102,6 +103,6 @@ constraints.flinkVersions.iceberg-0.13=1.14
 # Presto version restrictions - for Presto 0.274 --> only Iceberg 0.13
 constraints.prestoVersions.iceberg-0.13=0.274,0.275
 # Disable Presto for Iceberg 0.14 until there's a Presto release supporting Iceberg 0.14
-constraints.prestoVersions.iceberg-0.14=0.999
-# There's currently no Presto version that's compatible with Iceberg >= 0.14
-constraints.prestoVersions.iceberg-999.99=99.999
+constraints.prestoVersions.iceberg-0.14=0.276
+# Presto 0.276 uses Iceberg 0.14
+constraints.prestoVersions.iceberg-999.99=0.276
```

Comment:
what does the 999.99 stand for here?
maybe add a comment since the comment above imo doesnt really explain it, it seems like its explaining the previous line instead?